.github/workflows/build-push-image.yml

```diff
@@ -63,6 +67,11 @@ jobs:
       - uses: actions/checkout@v5
         with:
           fetch-depth: 0
+      
+      - name: Update the contents of the inner container if asked-for
+        if: ${{ inputs.update_container }}
+        run: |
+          echo "# A comment to generate a new container hash" >> dangerzone/conversion/doc_to_pixels.py
```

Comment:
I would add a random value here, else the hash will be consistently the same. I believe we may have issues in subsequent runs if the hash of the updated images is the same, but I'm not sure.